fleet-management-service/src/main/java/cz/fi/muni/pa165/service/VehicleServiceImpl.java

```diff
@@ -45,6 +46,17 @@ public double getTotalKilometrage(long vehicleId) {
         return result;
     }
 
+    public List<Vehicle> findActiveVehicles() {
+        List<Vehicle> all = vehicleDao.findAll();
+        List<Vehicle> active = new ArrayList<>();
+        for (Vehicle v : all) {
+            if (v.getActive()==null || v.getActive()) {
```

Comment:
if value is null or true it will add vehicle ... Why null value in vehicle mean active vehicle ?